lib/plugins/teams.js

```diff
@@ -1,10 +1,12 @@
 // NOTE: The APIs needed for this plugin are not supported yet by GitHub Apps
 // https://developer.github.com/v3/apps/available-endpoints/
```

Comment:
I think this warning can go away now because the endpoints are available now because this endpoint is supported by GitHub Apps